docs/zap-report.html

```diff
@@ -395,7 +395,7 @@ <h3>Alert Detail</h3>
   
   
 <tr bgcolor="#e8e8e8">
-<td width="20%">Instances</td><td width="80%">58</td>
+<td width="20%">Instances</td><td width="80%">56</td>
```

Comment:
I am using this file as a place to attach PR-level feedback. The comments on this file do not apply to the lines of code in question or the even the file.